### DISCUSSION TITLE
ADA_PROJECT_PATH in printenv?

### DISCUSSION THREAD

**Discussion by pjljvandelaar** [alire-project/alire#1196] [Q&A] [answered]
Hi Alire developers,

The documentation of [gnat project manager](https://docs.adacore.com/gprbuild-docs/html/gprbuild_ug/gnat_project_manager.html) describes  `ADA_PROJECT_PATH`. Yet, when I execute `alr printenv` this value is NOT described.
Is this an ommission? If not, why is it excluded?

Greetings,
   Pierre

**Comment by mosteo** [comment]
Alire uses `GPR_PROJECT_PATH` which to my knowledge is equivalent.

**Comment by Fabien-Chouteau** [comment, marked as answer]
See https://docs.adacore.com/gprbuild-docs/html/gprbuild_ug/gnat_project_manager.html#importing-projects:
```
ADA_PROJECT_PATH is used for compatibility, it is recommended to use GPR_PROJECT_PATH_FILE or GPR_PROJECT_PATH.
```
